docs/Installation.md

```diff
@@ -214,6 +220,56 @@ service apache24 restart
 Use the procedure for installation on [Debian](#2-debian).
 
 
+## Serving the GUI and API from a custom base url
+
+In some cases you may want to customize the application to change the base url
+from wich the GUI is served.
+
+To achieve that you will either need to change the `<base>` in the `index.html`
+or build the application yourself.
```

Comment:
It should be made clearer that updating `index.html` and custom build, respectively, are two alternatives. In both cases the web server configuration has to be updated from the default one.

Reply:
I removed the custom build alternative